release-notes.md

```diff
@@ -3,6 +3,65 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.16.0]
+
+# 1.16.0 (November 16 2022)
+
+#### New features & improvements
+
+* Android 13 support
+* Anbox now provides a serial id derived from the container ID to satisfy Android's conformance requirements.
+* AMD and Intel GPU support switched to be based on Vulkan and utilizing [ANGLE](https://chromium.googlesource.com/angle/angle) as the EGL/OpenGL ES frontend driver.
+* Mesa for AMD and Intel GPUs has been updated to the [22.2.2](https://docs.mesa3d.org/relnotes/22.2.2.html) release.
+* Split of Andorid's system and vendor directories.
+* New bug report utility inside the Anbox container.
+* Various WebRTC streaming improvements.
+* Introduction of the WebRTC data proxy server to enable full-duplex bidrectional data transmission between Android and a WebRTC based client.
+* Full Ubuntu 22.04 for regular and appliance deployment.
+* AMS allows configuring a custom APT mirror for all containers.
+* A new AMS charm configuration `use_network_acl` has been added to allow for experimental use of [LXD network ACLs](https://linuxcontainers.org/lxd/docs//howto/network_acls/) <!-- wokeignore:rule=master --> instead of the AMS node controller to restrict network access of Anbox containers.
```

Comment:
```suggestion
* Added a new AMS charm configuration `use_network_acl` to allow for experimental use of [LXD network ACLs](https://linuxcontainers.org/lxd/docs/latest/howto/network_acls/) instead of the AMS node controller to restrict network access of Anbox containers.
```